src/components/Mailbox.vue

```diff
@@ -351,7 +369,9 @@ export default {
 				break
 			case 'del':
 				logger.debug('deleting', { env })
-				this.onDelete(env.databaseId)
+				if (e.srcKey === 'del' && this.hasDeleteAcl()) {
+					this.onDelete(env.databaseId)
+				}
 				try {
 					await this.$store.dispatch('deleteThread', {
```

Comment:
this still executes, even with lacking ACL rights, doesn't it?

Reply:
Here i noticed a bug, the message is gone from the list giving you the impression that is deleted but after reload message it is still there.